Formula/haskell-language-server.rb

```diff
@@ -0,0 +1,44 @@
+class HaskellLanguageServer < Formula
+  desc "Integration point for ghcide and haskell-ide-engine. One IDE to rule them all"
+  homepage "https://github.com/haskell/haskell-language-server"
+
+  url "https://github.com/haskell/haskell-language-server.git",
+    tag:      "1.0.0",
+    revision: "4cd1cf934638881e52b3eba9f70157a4b799c0e9"
+
+  license "Apache-2.0"
+
+  head "https://github.com/haskell/haskell-language-server.git"
```

Comment:
```suggestion
  url "https://github.com/haskell/haskell-language-server.git",
      tag:      "1.0.0",
      revision: "4cd1cf934638881e52b3eba9f70157a4b799c0e9"
  license "Apache-2.0"
  head "https://github.com/haskell/haskell-language-server.git"
```
Let's follow other formulae here. (Mind the additional spacing before `tag:` and `revision:`. This is to align the arguments of `url`.)